pages/common/micro.md

```diff
@@ -0,0 +1,28 @@
+# micro
+
+> Micro is a terminal-based text editor that aims to be easy to use and intuitive, while also taking advantage of the full capabilities of modern terminals.
+> You can use your keyboard, but also your mouse to navigate and/or select text.
+
+- Open a file:
+
+`micro {{file}}`
+
+- Cut the entire line:
+
+`<Ctrl>k`
```

Comment:
Could you update the keyboard shortcuts to use the standard format?

e.g. `Ctrl + K`

See #1354 and #1356 for more details.